scripts/shelley-from-scratch/mkfiles.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
+
+set -e
+#set -x
+
+ROOT=example
+
+BFT_NODES="node-bft1 node-bft2"
+BFT_NODES_N="1 2"
+NUM_BFT_NODES=2
+
+POOL_NODES="node-pool1"
+
+ALL_NODES="${BFT_NODES} ${POOL_NODES}"
+
+if ! mkdir "${ROOT}"; then
+  echo "The ${ROOT} directory already exists, please move or remove it"
+  exit
+fi
+
+# copy and tweak the configuration
+cp configuration/defaults/byron-mainnet/configuration.yaml ${ROOT}/
+sed -i ${ROOT}/configuration.yaml \
+    -e 's/Protocol: RealPBFT/Protocol: TPraos/' \
+    -e 's/minSeverity: Info/minSeverity: Debug/'
+
+# Set up our template
+cardano-cli shelley genesis create --genesis-dir ${ROOT}
+
+# Then edit the genesis.spec.json ...
+
+SUPPLY=1000000000
+
+# We're going to use really quick epochs (300 seconds), by using short slots 0.2s
+# and K=10, but we'll keep long KES periods so we don't have to bother
+# cycling KES keys
+sed -i ${ROOT}/genesis.spec.json \
+    -e 's/"slotLength": 1/"slotLength": 0.2/' \
```

Comment:
```suggestion
    -e 's/"slotLength": 1/"slotLength": 0.1/' \
```

Reply:
I've been able to run this at 0.1 and then it can be tested twice as fast.

Reply:
It makes me nervous. 3min per epoch is pretty quick already :-)